frontend/apps/filemanager/filemanager.lua

```diff
@@ -114,74 +104,26 @@ end
 
 function FileManager:setupLayout()
     self.show_parent = self.show_parent or self
-    local icon_size = Screen:scaleBySize(DGENERIC_ICON_SIZE)
-    local home_button = IconButton:new{
-        icon = "home",
-        width = icon_size,
-        height = icon_size,
-        padding = Size.padding.default,
-        padding_right = Size.padding.large,
-        padding_bottom = 0,
-        callback = function()
-            self:goHome()
-        end,
-        hold_callback = function() self:setHome() end,
-    }
-
-    self.plus_button = IconButton:new{
-        icon = "plus",
-        width = icon_size,
-        height = icon_size,
-        padding = Size.padding.default,
-        padding_left = Size.padding.large,
-        padding_bottom = 0,
-        callback = function() self:onShowPlusMenu() end,
-    }
-
-    self.path_text = TextWidget:new{
-        face = Font:getFace("xx_smallinfofont"),
-        text = BD.directory(filemanagerutil.abbreviate(self.root_path)),
-        max_width = Screen:getWidth() - 2*Size.padding.large,
-        truncate_left = true,
+    self.title_bar = TitleBar:new{
+        fullscreen = "true",
+        align = "center",
+        title = self.title,
+        title_top_padding = Screen:scaleBySize(6),
```

Comment:
Was the `title_top_padding` we does compute if none provided not good ? Or just different from the previous look ? And if different, not good enough ?
The computation also takes into account button_padding that you set to Screen:scaleBySize(5).
How is it if you provide none, or just one button_padding?
Why is `button_padding = Screen:scaleBySize(11), -- fine to keep exit/cross icon diagonally aligned with screen corners ` not good ? Because this was set to look good with `left_icon_size_ratio = 0.6` only, and is bad with `= 1` ?

Just asking, i'm fine with this as-is. I had done enough maths, I don't need more :)